oppia/oppia-android
# Fix #387: Concept card linking

## Explanation

Fixes #387 
This PR adds the concept card linking inside exploration player. This has been done by using #422 
Thanks to @BenHenning for #422 

TAG information (as confirmed with @seanlip ):

`<oppia-noninteractive-skillreview skill_id-with-value="<[skillId]>" text-with-value="<[linkText]>"></oppia-noninteractive-skillreview>`

## How to test this:
Approach 1. Make `HtmlParserTestActivity` as your launcher activity.
Approach 2. Start `Meaning of Equal Parts` exploration. First answer is `D. last option`. In second state, answer 3/2 and you should see a `refresher lesson` text.

## Pending Issues:
- [ ] LiTagHandler is not getting used.

## Checklist
